account/watcher/controller.go

```diff
@@ -67,8 +66,8 @@ type Watcher struct {
 }
 
 // New instantiates a new chain watcher backed by the given config.
-func New(cfg *Config) *Watcher {
-	return &Watcher{
+func NewController(cfg *Config) *controller { // nolint:golint
```

Comment:
Ah, I guess that's one downside of having a non-exported implementation. But I think that's okay. Or we could break the "return structs" philosophy here and return the interface instead of silencing the linter. Either way is fine with me, your call.